v0/destinations/active_campaign/data/ACIdentify.json

```diff
@@ -0,0 +1,49 @@
+[
+  {
+    "destKey": "email",
+    "sourceKeys": [
+        "traits.email",
+        "context.traits.email",
+        "properties.email"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "firstName",
+    "sourceKeys": [
+        "traits.firstName",
+        "context.traits.firstName",
+        "properties.lastName"
+      ],
+    "required": false
+  },
+  {
+    "destKey": "lastName",
+    "sourceKeys": [
+        "traits.lastName",
+        "context.traits.lastName",
+        "properties.lastName"
+      ],
+    "required": false
+  },
```

Comment:
There is a method for extracting `firstName` and `lastName` from the `traits` in `util/index.js`. Can we use that to keep all the destination logics similar?